susemanager-utils/testing/docker/head/suma-head-base/add_packages.sh

```diff
@@ -74,3 +74,8 @@ zypper --non-interactive in ant \
 # Packages for easier debugging
 zypper --non-interactive in vim less
 
+# Packages for Javascript tests
+zypper --non-interactive in nodejs8 npm8
+
+# Install flow-bin globally
+npm_config_prefix=/usr npm install -g flow-bin@0.73
```

Comment:
Hmm, the base container is getting bigger and bigger.
I would  prefer a new container on top of base or postgresql (if this is really needed).
A special suma-<ver>-eslint or suma-<ver>-nodejs container.